File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/accounting/transactions/shared/transaction-link.model.ts

export class TransactionLink {
    public AccountingTxnLinkId: number = 0;
    public TransactionId: number = 0;
    public ReferenceId: string = null;
    public TransferStatus: number = 0;//this for is cash, credit or credit return status i.e. NOT-0,Cash-1,Credit-2
    public TransactionItemId: number = 0;
}

File: Code/Components/DanpheEMR.ServerModel/AccountingModels/TransactionLinkModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel
{
    public class TransactionLinkModel
    {
        [Key]
        public int AccountingTxnLinkId { get; set; }
        public int? TransactionId { get; set; }
        public string ReferenceId { get; set; }      
        [NotMapped]
        public int? TransferStatus { get; set; }
    }  
}
